packages/marathon/buildinfo.json

```diff
@@ -4,8 +4,8 @@
   ],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.9.124-e04033c6f/marathon-1.9.124-e04033c6f.tgz",
-    "sha1": "5b8933377a6b34b2dee2d692b0b9dc7eb33cd62d"
+    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.10.7-e52782ed7/marathon-1.10.7-e52782ed7.tgz",
```

Comment:
How did you get `1.10.7` when `master` is `1.10.4`?

Reply:
it's a snapshot